### PR TITLE
Fixed evidence bags overlays

### PR DESCRIPTION
**Pull request by JMWTurner**
Previously, the item they contain wouldn't appear in your inventory because the overlay was on a much lower plane, and using the bag on a forensic computer to transfer the item wouldn't remove the overlay.

### CODE REVIEW

**Comment by clusterfack** [comment]
Would have been nice if you fixed them and absolute pathed them in different commits, I don't know where to trawl to see your fix.

File: code/modules/detectivework/detective_work.dm [hunk 1]
```diff
@@ -169,7 +169,7 @@ var/const/FINGERPRINT_COMPLETE = 6	//This is the output of the stringpercent(pri
 				if(istype(I, /obj/item/weapon/evidencebag))
 					scanning = I.contents[1]
 					scanning.forceMove(src)
-					I.overlays -= scanning
+					I.overlays.len = 0
```
Comment (JMWTurner) [inline code/modules/detectivework/detective_work.dm:172]: @clusterfack

File: code/modules/detectivework/evidence.dm [hunk 2]
```diff
@@ -47,6 +47,7 @@
 	I.pixel_x = 0		//then remove it so it'll stay within the evidence bag
 	I.pixel_y = 0
 	var/image/img = image("icon"=I, "layer"=FLOAT_LAYER)	//take a snapshot. (necessary to stop the underlays appearing under our inventory-HUD slots ~Carn
+	img.plane = HUD_PLANE
```
Comment (JMWTurner) [inline code/modules/detectivework/evidence.dm]: @clusterfack

**Comment by JMWTurner** [comment]
Actually I didn't change anything in detective_scanner.dm, I only absolute pathed it.

File: code/modules/detectivework/evidence.dm (hunk 2 above)
Comment (ghost) [reply code/modules/detectivework/evidence.dm]: Why hud plane and not float plane?

Comment (JMWTurner) [reply code/modules/detectivework/evidence.dm]: > FLOAT_PLANE = -32767

er, wouldn't that be counter productive?

Comment (clusterfack) [inline code/modules/detectivework/evidence.dm]: You have the byond reference at your fingertips and yet you choose to not use it and instead doubt

Comment (JMWTurner) [reply code/modules/detectivework/evidence.dm]: > The special value FLOAT_PLANE can be used for images and overlays, to take on the plane of the parent atom. Whenever an icon or icon_state is used directly as an overlay, this is its plane. 

Well, the more you know

**Pull request merged**